Vagrantfile

```diff
@@ -15,5 +15,13 @@ Vagrant.configure("2") do |config|
   end
   config.vm.provision "ansible_local" do |ansible|
       ansible.playbook = "./test/playbook.yml"
+
+      # default mode `dev`
+      ansible.extra_vars = {
+        mode: 'dev'
+      }
+
+      # use to override default mode (e.g. test mode)
+      ansible.raw_arguments = Shellwords.shellsplit(ENV['ANSIBLE_ARGS']) if ENV['ANSIBLE_ARGS']
```

Comment:
What is happening here?

Reply:
From stdin. Look at Makefile target: test

References:
- [Passing command line arguments into Vagrant to configure Ansible Provisioning](https://gist.github.com/phantomwhale/9657134)
- [Running Ansible provisioning by passing arguments in Vagrant](http://www.inanzzz.com/index.php/post/wfj9/running-ansible-provisioning-by-passing-arguments-in-vagrant)